test/e2e/cmd/run/run.go

```diff
@@ -297,29 +297,71 @@ func (h *helper) createRoles() error {
 	return h.kubectlApplyTemplateWithCleanup("config/e2e/roles.yaml", h.testContext)
 }
 
-func (h *helper) installOperator() error {
-	log.Info("Installing the operator")
-	values, err := h.renderTemplate("config/e2e/helm-values.yaml", h.testContext)
-	if err != nil {
-		return fmt.Errorf("failed to generate Helm values: %w", err)
+func (h *helper) installE2EOperator() error {
+	log.Info("Installing the E2E operator")
+
+	installCRDs := false
```

Comment:
This took me a while to wrap my head around. Initially I thought that we are always testing the 1.4.0 CRDs and not the ones of the version under test until I realised that we just setting the Docker image but the CRDs and other manifests are actually based on the version under test. I think maybe a comment somewhere would be good to make the casual reader aware of this.

Reply:
Yes, it's not ideal but it's not much different from how it used to be either. Since CRDs are global, we have to install the current version in order to properly test the operator. That means not being able to use the all-in-one.yaml for 1.4.0 because it would overwrite the CRDs. We could use Helm proper, but that means having to install Helm into the CI image, syncing with the Elastic Helm repo etc. I didn't think the additional complexity was worth it given how complicated the whole thing is already.